src/lib/resources.ts

```diff
@@ -55,18 +54,52 @@ export class Vaults extends OPResource {
         return ObjectSerializer.deserialize(data, "Array<Item>");
     }
 
+    /**
+     * Get metadata about a single vault
+     *
+     * @param vaultQuery
```

Comment:
The parameter name may not be self explanatory. Adding some extra documentation would help here:
```suggestion
     * @param vaultQuery - the Vault's title or ID
```

Reply:
done